CIP-0000.md

```diff
@@ -0,0 +1,198 @@
+### `discussions-to` header
+While a CIP is a draft, a discussions-to header will indicate the mailing list or URL where the CIP is being discussed. As mentioned above, examples for places to discuss your CIP include the Celo Discord , an issue in this repo or in a fork of this repo.
+No discussions-to header is necessary if the CIP is being discussed privately with the author.
+As a single exception, discussions-to cannot point to GitHub pull requests.
+### `type` header
+The type header specifies the type of CIP: Standards Track, Meta, or Informational. If the track is Standards please include the subcategory (core, networking, interface, or CRC).
+### `category` header
+The category header specifies the CIP’s category. This is required for standards-track CIPs only.
+### `created` header
+The created header records the date that the CIP was assigned a number. Both headers should be in yyyy-mm-dd format, e.g. 2001-08-14.
+### `updated` header
+The updated header records the date(s) when the CIP was updated with “substantial” changes. This header is only valid for CIPs of Draft and Active status.
+### `requires` header
+CIPs may have a requires header, indicating the CIP numbers that this CIP depends on.
+### `superseded-by` and `replaces` headers
+CIPs may also have a superseded-by header indicating that a CIP has been rendered obsolete by a later document; the value is the number of the CIP that replaces the current document. The newer CIP must have a replaces header containing the number of the CIP that it rendered obsolete.
+
+## Auxiliary Files
+Images, diagrams and auxiliary files should be included in a subdirectory of the `assets` folder for that CIP as follows: `assets/cip-N` (where **N** is to be replaced with the CIP number). When linking to an image in the CIP, use relative links such as `../assets/cip-1/image.png`.
+
+
+# Transferring CIP Ownership
+It occasionally becomes necessary to transfer ownership of CIPs to a new champion. In general, we’d like to retain the original author as a co-author of the transferred CIP, but that’s really up to the original author. A good reason to transfer ownership is because the original author no longer has the time or interest in updating it or following through with the CIP process, or has fallen off the face of the ‘net (i.e. is unreachable or isn’t responding to email). A bad reason to transfer ownership is because you don’t agree with the direction of the CIP. We try to build consensus around a CIP, but if that’s not possible, you can always submit a competing CIP.
+If you are interested in assuming ownership of a CIP, send a message asking to take over, addressed to both the original author and the CIP editor. If the original author doesn’t respond to email in a timely manner, the CIP editor will make a unilateral decision.
+
+# CIP Editors
+The current CIP editors are:
+`* James Prestwich (@prestwich)`
+`* Yaz Khoury (@YazzyYaz)`
```

Comment:
```suggestion
`* Yaz Khoury (@YazzyYaz)`
`* Amy Slawson (@aslawson)`
```

Reply:
Amy has stewarded a lot of the governance process so far -- @aslawson , happy to be involved?

Also would love non-cLabs representation here, esp someone outside of North America

Reply:
Yeah, happy to add Amy if she wants to be an editor. Also would love some non-cLabs representation if anyone in the community would like to be involved.